client/components/AuthForm.jsx

```diff
@@ -1,19 +1,18 @@
 import React from 'react';
-import AuthStore from '../stores/AuthStore';
-import AuthActions from '../actions/AuthActions';
+import AuthStore from 'stores/AuthStore';
+import AuthActions from 'actions/AuthActions';
 import connectToStores from 'alt/utils/connectToStores';
 import classnames from 'classnames';
 
-const displayName = 'AuthForm';
-const propTypes = {
-    isWaiting: React.PropTypes.bool,
-    isLoggedIn: React.PropTypes.bool,
-    username: React.PropTypes.string,
-    error: React.PropTypes.string
-};
-
 @connectToStores
 export default class AuthForm extends React.Component {
+    static propTypes = {
+        isWaiting: React.PropTypes.bool,
+        isLoggedIn: React.PropTypes.bool,
+        username: React.PropTypes.string,
+        error: React.PropTypes.string
+    };
```

Comment:
Nice. I like this as a static within the class instead of declaring it after the fact. I couldn't get lint to pass before when I tried that, but I just did it now and it seemed to work, so I'm going to go ahead and change all of those to be `static` instead.